Ubuntu AWS and Ubuntu pro pages

anthonydillon opened the pull request:
## Done
Build of the AWS and AWS pro pages

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to /aws and check it matches the [copy doc](https://docs.google.com/document/d/1lkH-MSPmgjCMP4gI0FW29YIKVDvBuddxA63iQhmsYIU/edit?ts=5de63dcb#) and the [design](https://usercontent.irccloud-cdn.com/file/mjMJMvDg/image.png)
- Go to /aws/pro and check it matches the [copy doc](https://docs.google.com/document/d/1pLRWJZSpE04nYt-yHBtCjxGI7VZsigcP2UJIBmR8i-0/edit#)
- Go to /aws/contact-us and check to works
- Check the thank you page at /aws/thank-you looks ok

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2030
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2032

webteam-app commented:
Starting demo at: https://ubuntu-com-canonical-web-and-design-pr-6216.run.demo.haus/

anasereijo commented:
@anthonydillon same comments as Spencer + these ones:

Overview:

- [ ] make the AWS logo 200 px wide

- [ ] "Optimise your cloud journey with Canonical": make it 6 cols and quote 5

- [ ] quote looks very small comparing to design: make it `class="p-pull-quote"`

anthonydillon commented:
@spencerbygraves @anasereijo I have addressed all the points I can for now.

>  make the "* For 18.04 LTS onwards. The maintenance period for Ubuntu Pro 14.04 LTS and 16.04 LTS is 8 years." 

I assume you mean smaller

>  The piechart is now a graph

I have removed the graph for now

anasereijo commented:
@anthonydillon looks good!!

The quote on /pro page looks a bit weird, and should the quote marks be orange as usual? See below:

desktop:
<img width="1073" alt="Screenshot 2019-12-04 at 12 12 24" src="https://user-images.githubusercontent.com/36884067/70141686-83777580-168f-11ea-825b-aa233f0549ee.png">

mobile:
<img width="412" alt="Screenshot 2019-12-04 at 12 12 45" src="https://user-images.githubusercontent.com/36884067/70141697-896d5680-168f-11ea-849e-1c0286b1a818.png">

sowasred2012 commented on templates/aws/index.html:
```diff
@@ -0,0 +1,143 @@
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
```
Don't think this needs to be a `row`
```suggestion
  <div class="u-fixed-width">
```

anthonydillon replied:
Same as below

sowasred2012 commented on templates/aws/pro.html:
```diff
@@ -0,0 +1,267 @@
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
```
```suggestion
  <div class="u-fixed-width">
```

anthonydillon replied:
I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

sowasred2012 commented on templates/aws/pro.html:
```diff
@@ -0,0 +1,267 @@
+</section>
+
+<section class="p-strip">
+  <div class="row">
```
```suggestion
  <div class="u-fixed-width">
```

anthonydillon replied:
I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

sowasred2012 commented on templates/aws/pro.html:
```diff
@@ -0,0 +1,267 @@
+</section>
+
+<section class="p-strip">
+  <div class="row">
```
```suggestion
  <div class="u-fixed-width">
```

anthonydillon replied:
I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

sowasred2012 commented on templates/aws/pro.html:
```diff
@@ -0,0 +1,267 @@
+</section>
+
+<section class="p-strip">
+  <div class="row">
```
```suggestion
  <div class="u-fixed-width">
```

anthonydillon replied:
I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

sowasred2012 commented on templates/aws/pro.html:
```diff
@@ -0,0 +1,267 @@
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821V7QJP" class="p-link--external">14.04 LTS</a></h3>
+    </div>
+  </div>
+  <div class="row">
```
```suggestion
  <div class="u-fixed-width">
```

anthonydillon replied:
I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

sowasred2012 commented on templates/shared/forms/interactive/_aws.html:
```diff
@@ -0,0 +1,183 @@
+
+    <div class="js-pagination js-pagination--4 u-hide">
+      <div class="row u-no-padding">
+        <div class="u-equal-height">
```
Could this class be added to the div above? `col-`s should be direct children of `row`s.

anthonydillon replied:
Good point

anthonydillon commented:
@anasereijo @sowasred2012 addressed your comments

sowasred2012 commented on templates/aws/pro.html:
```diff
@@ -0,0 +1,275 @@
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 16.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
```
The copy doc has "The most commonly deployed Ubuntu LTS on the cloud today, Ubuntu 16.04 LTS released in 2016 with the 4.4 kernel, and is covered with Ubuntu Pro through 2024." for this section.

sowasred2012 commented on templates/aws/pro.html:
```diff
@@ -0,0 +1,275 @@
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 14.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
```
The copy doc has "Ubuntu 14.04 LTS shipped in 2014 with the 3.13 kernel. Ubuntu Pro extends our security coverage to 2022, giving you plenty of time to prepare and upgrade."

anasereijo commented:
nice

anthonydillon commented:
@sowasred2012 thanks updated
